doc/installation_instructions/troubleshooting/custom_backends.rst

```diff
@@ -0,0 +1,41 @@
+Custom ML backends
+------------------
+
+The ML backends (Torch, ONNX Runtime, and Tensorflow) and their associated
```

Comment:
supported based on the intended device (CPU, ROCm, CUDA-11, or CUDA-12) - I think the only device in the list is CPU right? Maybe say device or platform